rules/windows/privilege_escalation_group_policy_privileged_groups.toml

```
@@ -0,0 +1,93 @@
+[metadata]
+creation_date = "2021/11/08"
+maturity = "production"
+updated_date = "2021/11/08"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects the modification of Group Policy Object Attributes (for the first time) to add privileges to user accounts or use them to
```

is this a more accurate way to say it

```suggestion
This rule detects the first occurrence of a modification to Group Policy Object Attributes to add privileges to user accounts or use them to
```

---

Yeah, sounds better